docs/hikey.md

```diff
@@ -57,6 +57,9 @@ before building, see [UART in hikey.mk].
 Just follow the "Get and build the solution" in the [README.md] file. The `make
 flash` step will tell you how you should set the jumpers on the board.
 
+**NOTE:** If you're using the Kirin 620 or Hikey from CircuitCo, please set
```

Comment:
Yes, please write "'If you're using a Hikey board with a 4GB flash such as the older boards manufactured by CircuitCo" or something like that.